project_event/wizard/project_event_wizard.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding:utf-8 -*-
+# @ 2018 Savoir-failre Linux
+# License LGPL-3.0 or Later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models, _
+
+
+class ProjectEventWizard(models.TransientModel):
+    _name = "project.event.wizard"
+    _description = "Wizard Event Creation from Template"
+
+    template_id = fields.Many2one(
+        'event.template',
+        string='Event Template',
+    )
+    name = fields.Char(
+        string='Event Title',
+    )
+    event_resp_id = fields.Many2one(
+        'res.partner',
+        string='Responsible',
+    )
+    event_notes = fields.Text(
+        string='Notes',
+    )
+    activity_line_ids = fields.One2many(
+        'activity.template.line.wiz',
+        'event_wiz_id',
```

Comment:
@rimbendhaou: Not good string "event_wiz_id". My comment or suggestion is "event_wizard_id" or "project_event_wizard_id"